src/database/DataTypes/Stocktake.js

```diff
@@ -285,7 +292,10 @@ export class Stocktake extends Realm.Object {
    */
   addItemsFromProgram(database) {
     if (!this.program) return false;
-    this.setItemsByID(database, this.program.items.map(masterListItem => masterListItem.item.id));
+    this.setItemsByID(
+      database,
+      this.program.items.map(masterListItem => masterListItem.item.id)
+    );
```

Comment:
Looks like some prettier changes sneaked into the commit, unrelated to logic changes!